src/app/search-results-summary-report/search-results-summary-report.component.ts

```diff
@@ -475,38 +475,23 @@ export class SearchResultsSummaryReportComponent implements OnInit {
           recordStatus = 'Incomplete';
         }
         row.push({ text: recordStatus, alignment: 'left', fontSize: 10 });
-        // TODO: need to come back and fix this. it's a number. Maybe need to have organization_string added to event? Or maybe just use organization service
-        if (elData.organizations) {
+        const organizationCell = new Array();
+        if (elData.organizations !== null && elData.organizations !== undefined) {
 
-          // row.push(elData.organizations[0].toString());
-        } else {
-          // row.push('');
-        }
-        const orgCell = new Array();
-        if (elData.organizations !== undefined) {
           const semiColon = elData.organizations.length - 1;
           for (let i = 0; i < elData.organizations.length; i++) {
-            let organization;
-            for (const orgKey in this.orgs) {
-              if (this.orgs[orgKey].id === elData.organizations[i]) {
-                organization = this.orgs[orgKey].name;
-              }
-            }
             if (i !== semiColon) {
-              orgCell.push({ text: organization + ';\n', alignment: 'left' });
```

Comment:
Why delete the new line?

Reply:
@lprivette this was from an older version of the logic that built the org name list after running it through a matching function to get the name. With the server update, the whole org object comes back so we can just access that and no need for the matching. this line is replaced with a similar line doing effectively the same thing (line 484 on the new side)